src/Bridge/Laravel/Jobs/LogHttpRequest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+declare(strict_types=1);
+
+namespace LoyaltyCorp\Auditing\Bridge\Laravel\Jobs;
+
+use LoyaltyCorp\Auditing\DataTransferObjects\LogLine;
+use LoyaltyCorp\Auditing\Interfaces\Services\LogWriterInterface;
+
+class LogHttpRequest extends Job
+{
+    /**
+     * @var \LoyaltyCorp\Auditing\DataTransferObjects\LogLine
+     */
+    private $logLine;
```

Comment:
Do we need to ensure that the LogLine can be serialized?

Reply:
Yes, considering the base job class has the `ShouldQueue` trait.
I added a test to validate `LogLine` is serializable - https://github.com/loyaltycorp/auditing/pull/7/files#diff-aaa459522dd21deb1343b58140dee922R24

It can be removed if we find that we only want to do it synchronously